libs/shared/ui/src/lib/styles/components/button.scss

```diff
@@ -38,11 +38,23 @@
   @apply bg-transparent text-brand-500 hover:bg-transparent hover:text-brand-700;
 }
 
+.btn--alt {
+  @apply bg-transparent text-text-400 hover:bg-element-light-lighter-400 hover:text-brand-500;
+}
+
+.btn--dark {
+  @apply bg-element-light-darker-100 text-text-100 hover:bg-element-light-darker-200;
+}
+
 // state
 .btn--disabled {
   @apply cursor-default pointer-events-none bg-brand-100 text-brand-300 border-0;
 }
 
+.btn--active {
```

Comment:
Why do we use a class "active" for the global btn?
The "active" of each button is different following its basic design